features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/org.wso2.carbon.identity.application.authentication.framework.server.feature.default.json

```diff
@@ -81,43 +83,51 @@
 
   "authentication.authenticator.oauth_request_path.name": "OAuthRequestPathAuthenticator",
   "authentication.authenticator.oauth_request_path.enable": true,
+  "authentication.authenticator.oauth_request_path.parameters.Tags": "Request path",
 
   "authentication.authenticator.saml.name": "SAMLSSOAuthenticator",
   "authentication.authenticator.saml.enable": true,
+  "authentication.authenticator.saml.parameters.Tags": "SAML",
 
   "authentication.authenticator.oidc.name": "OpenIDConnectAuthenticator",
   "authentication.authenticator.oidc.enable": true,
   "authentication.authenticator.oidc.parameters.enableIatValidation": true,
   "authentication.authenticator.oidc.parameters.iatValidityPeriod": "300",
+  "authentication.authenticator.oidc.parameters.Tags": "OIDC",
 
   "authentication.authenticator.google.name": "GoogleOIDCAuthenticator",
   "authentication.authenticator.google.enable": true,
   "authentication.authenticator.google.parameters.GoogleTokenEndpoint": "https://accounts.google.com/o/oauth2/token",
   "authentication.authenticator.google.parameters.GoogleAuthzEndpoint": "https://accounts.google.com/o/oauth2/auth",
   "authentication.authenticator.google.parameters.GoogleUserInfoEndpoint": "https://www.googleapis.com/oauth2/v3/userinfo",
+  "authentication.authenticator.google.parameters.Tags": "Social Login",
 
   "authentication.authenticator.microsoft_live.name": "MicrosoftWindowsLive",
   "authentication.authenticator.microsoft_live.enable": true,
   "authentication.authenticator.microsoft_live.parameters.AuthTokenEndpoint": "https://login.live.com/oauth20_token.srf",
   "authentication.authenticator.microsoft_live.parameters.AuthnEndpoint": "https://login.live.com/oauth20_authorize.srf",
   "authentication.authenticator.microsoft_live.parameters.UserInfoEndpoint": "https://apis.live.net/v5.0/me?access_token=",
+  "authentication.authenticator.microsoft_live.parameters.Tags": "Social Login,OIDC",
```

Comment:
is OIDC required here? asking this based on the tags defined for google

Reply:
Addressed with commit ad110cdff4ffcfe5766b31f1a09bebaf633730e6